adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -438,7 +442,7 @@ private void LogReturnedToken(AdalResult result)
 
         protected void ValidateAuthorityType()
         {
-            if (!SupportADFS && Authenticator.AuthorityType == AuthorityType.ADFS)
+            if (!SupportADFS && Authenticator.AuthorityType == Instance.AuthorityType.ADFS)
```

Comment:
>Instance [](start = 63, length = 8)

?

Reply:
to not interfere with AuthorityType from Core

Reply:
Can't you use the namespace to be more specific instead? Also why do we not only have this in core? do we need to have this type in other places?

---
In reply to: [191035342](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1065#discussion_r191035342) [](ancestors = 191035342)